src/containers/app/navigation/menu/index.js

```diff
@@ -1,36 +1,61 @@
 import React from 'react';
-import { withRouter, useLocation } from 'react-router-dom';
-import PropTypes from 'prop-types';
+import { useHistory, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Menu } from 'antd';
+
 import {
   ROUTE_DASHBOARDS,
   ROUTE_PIPELINES,
   ROUTE_JOBS,
 } from 'config/routes';
-import styles from './index.module.css';
+import colors from 'styles/colors';
+
+const StyledMenu = styled(Menu)`
+  padding: 23px 9px 0 0;
+  &.ant-menu-vertical .ant-menu-item-selected {
+    background-color: ${colors.black};
+    color: ${colors.white};
+    cursor: default;
+  }
+  &.ant-menu-vertical > .ant-menu-item {
+    margin: 0;
+    height: auto;
+    line-height: 47px;
+  }
+`;
+
+const StyledMenuItem = styled(Menu.Item)`
+  padding-left: 16px 13px; 
+  font-size: 16px;
+  font-weight: 500;
+  text-transform: uppercase;
+  border-radius: 2px;
+  &, &:hover {
+    color: #404040;
+  }
+  &:hover {
+    background-color: ${colors.lightGray};
+  }
+`;
 
-const Menu = (props) => {
-  const { history } = props;
+const MainMenu = () => {
+  const history = useHistory();
   const navigate = (path) => (() => history.push(path));
   const location = useLocation();
   const path = location.pathname;
 
-  const dashboardsStyle = path.includes(ROUTE_DASHBOARDS) ? styles.selected : styles.unselected;
-  const pipelinesStyle = path.includes(ROUTE_PIPELINES) ? styles.selected : styles.unselected;
-  const jobsStyle = path.includes(ROUTE_JOBS) ? styles.selected : styles.unselected;
+  const selectedKeys = [];
```

Comment:
Ugghh.. this looks ugly. Let's see what we can do about this later. Good enough for now